janus_server/src/bin/aggregator.rs

```diff
@@ -1,101 +1,30 @@
-use anyhow::{anyhow, Context, Result};
+use anyhow::{Context, Result};
 use futures::StreamExt;
 use janus::time::RealClock;
 use janus_server::{
     aggregator::aggregator_server,
-    binary_utils::datastore,
+    binary_utils::{janus_main, BinaryOptions, CommonBinaryOptions},
     config::AggregatorConfig,
-    metrics::{install_metrics_exporter, MetricsExporterConfiguration},
-    trace::{cleanup_trace_subscriber, install_trace_subscriber, OpenTelemetryTraceConfiguration},
-};
-use std::{
-    fmt::{self, Debug, Formatter},
-    fs::File,
-    future::Future,
-    iter::Iterator,
-    path::PathBuf,
-    sync::Arc,
 };
+use std::{future::Future, iter::Iterator, sync::Arc};
 use structopt::StructOpt;
 use tracing::info;
 
-#[derive(StructOpt)]
+#[derive(Debug, StructOpt)]
 #[structopt(
     name = "janus-aggregator",
     about = "PPM aggregator server",
     rename_all = "kebab-case",
     version = env!("CARGO_PKG_VERSION"),
 )]
 struct Options {
```

Comment:
This struct (and the equivalent ones in `aggregation_job_driver.rs` and `aggregation_job_creator.rs` exist solely so we can provide a `#[structopt(...)]` block with the `name` and `about` values.